coordinator/src/services/communication/ebxml-request-builder.ts

```diff
@@ -39,10 +40,12 @@ class EbXmlRequest {
   }
 }
 
-export function addEbXmlWrapper(spineRequest: SpineRequest): string {
+export function addEbXmlWrapper(spineRequest: SpineRequest, logger: Logger): string {
   const cpaId = cpaIdMap.get(spineRequest.interactionId)
   if (!cpaId) {
-    throw new Error(`Could not identify CPA ID for interaction ${spineRequest.interactionId}`)
+    const errorString = `Could not identify CPA ID for interaction ${spineRequest.interactionId}`
```

Comment:
Would this message not be caught by the logs anyway (as part of the exception handler)?

Reply:
And if we are logging the message for security reasons, do we want something else as the message for exception ('Could not find the specified CPA ID?')

Reply:
Thrown errors aren't shown in the logs, so previously this was just erroring out with no logging message

Reply:
Not sure it's a security thing, it was more for myself so I could see why things crashed out at that point. I thought it could be a  useful reminder when we start adding the other interaction IDs, but I can remove if we don't think it's important